config/routes.rb

```diff
@@ -7,9 +7,14 @@
   get "up" => "rails/health#show", as: :rails_health_check
 
   # Defines the root path route ("/")
-  root 'bright_events#index'
+  root 'home#index'
 
   get 'home/' => 'home#index'
```

Comment:
these two routes should be reduced to only `root 'home#index', what do you think?

Reply:
done :+1: